### DISCUSSION TITLE
Is there a way to display another modal after sending one ?

### DISCUSSION THREAD

**Discussion by julesbsz** [Mateo-tem/discord-modals#94] [Q&A] [answered]
Hello everyone, I'm new to DiscordJS and I would like to know if it's possible, when sending a modal, to open directly a second one, then a third one etc.. ?
Thank you !

**Comment by Mateo-tem** [comment, marked as answer]
That should be called "Modal Pagination" or something like that. Well, this is not actually possible. Discord has Paginated Modals in plans ;)
